rhino.js

```diff
@@ -31,7 +30,7 @@ if (process.env.JAVA_HOME) {
 }
 
 function executeScript(scriptName) {
-    return child_process.execFileSync(javaCommand, ['-jar', 'rhino.jar', '-version', '200', scriptName], {
```

Comment:
can you elaborate on this change?

Reply:
Yes! From the release notes:
> The default language level is "VERSION_ES6". That means that modern JavaScript features supported by Rhino will work by default.

And the relevant PR: https://github.com/mozilla/rhino/pull/1594